.github/workflows/call-clear-cache.yml

```diff
@@ -23,18 +23,18 @@ on:
 jobs:
   cron-clear:
     if: github.event_name == 'schedule' || github.event_name == 'pull_request'
-    uses: Lightning-AI/utilities/.github/workflows/cleanup-caches.yml@v0.14.3
+    uses: Lightning-AI/utilities/.github/workflows/cleanup-caches.yml@v0.15.0
     with:
-      scripts-ref: v0.14.3
+      scripts-ref: v0.15.0
       dry-run: ${{ github.event_name == 'pull_request' }}
       pattern: "latest|docs"
       age-days: 7
 
   direct-clear:
     if: github.event_name == 'workflow_dispatch' || github.event_name == 'pull_request'
-    uses: Lightning-AI/utilities/.github/workflows/cleanup-caches.yml@v0.14.3
+    uses: Lightning-AI/utilities/.github/workflows/cleanup-caches.yml@v0.15.0
     with:
-      scripts-ref: v0.14.3
+      scripts-ref: v0.15.0
```

Comment:
```suggestion
      scripts-ref: v0.14.3
```

Reply:
this has breaking issue, need to be resolved
https://github.com/Lightning-AI/utilities/pull/371#issuecomment-3150211432